api/src/main/java/org/pmiops/workbench/config/WorkbenchConfig.java

```diff
@@ -141,6 +141,9 @@ public String freeTierBillingAccountName() {
   }
 
   public static class ServerConfig {
+    // Base URL for the webapp (e.g. client / ui service).
```

Comment:
I was a little surprised that we didn't have this in our backend config already. I think it's distinct enough from the API base URL (they're different App Engine services with different base paths) that it's worth including here. I'm open to alternative suggestions, though.